src/React.Router/ReactRouterComponent.cs

```diff
@@ -0,0 +1,141 @@
+﻿/*
+ *  Copyright (c) 2014-Present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+ using JavaScriptEngineSwitcher.Core;
+using Newtonsoft.Json;
+using React.Exceptions;
+
+namespace React.Router
+{
+	/// <summary>
+	/// Represents a React Router JavaScript component.
+	/// </summary>
+	public class ReactRouterComponent : ReactComponent
```

Comment:
I'm sorry but I'm quite confused.
I'm viewing the file now in vstudio with view whitespace.
I can see a random leading whitespace in front of one using statement and some weirdness with the javascript block that is about to be removed.
Aside from that it shows only tabs as indentation in this file.